charts/lightdash/Chart.lock

```diff
@@ -8,5 +8,8 @@ dependencies:
 - name: browserless-chrome
   repository: https://charts.sagikazarmark.dev
   version: 0.0.4
-digest: sha256:e67eebb04f4ad84367593b74cf5dd71a4ee8e5daede54549be6dfea396419064
-generated: "2023-01-11T20:11:55.264423Z"
+- name: scheduler
```

Comment:
can we call this `lightdash-scheduler` so it's obviously a lightdash-specific dependency